include/nuttx/wdog.h

```diff
@@ -29,7 +29,6 @@
 
 #include <nuttx/clock.h>
 #include <stdint.h>
-#include <queue.h>
```

Comment:
Seems now we need to add `#include <nuttx/queue.h>`  to `sched/wdog/wdog.h` as it was implicitly included till now

Reply:
Done.